tools/crm_resource_runtime.c

```diff
@@ -1321,10 +1321,19 @@ cli_resource_restart(pcmk__output_t *out, pe_resource_t *rsc, pe_node_t *node,
     }
 
     rsc_id = strdup(rsc->id);
-    lookup_id = clone_strip(rsc->id);
 
-    if ((pe_rsc_is_clone(rsc) || pe_rsc_is_clone(parent) || pe_bundle_replicas(rsc)) && host) {
-        stop_via_ban = true;
+    if (host) {
+        if (pe_rsc_is_clone(rsc) || pe_bundle_replicas(rsc)) {
+            stop_via_ban = true;
+            lookup_id = clone_strip(rsc->id);
```

Comment:
If rsc is a clone or has replicas (i.e. it is not a clone instance or a replica) then clone_strip() does nothing -- basically i think this and the next test are the same thing

Reply:
I have rearranged this code somewhat to reduce duplication, so we'll have to check this out on the next push.  I'm not sure how much it applies anymore.